packages/perseus/src/widgets/expression.tsx

```diff
@@ -356,6 +362,17 @@ export class Expression extends React.Component<Props, ExpressionState> {
         // TODO(scottgrant): This is a hack to remove the deprecated call to
         // this.isMounted() but is still considered an anti-pattern.
         this._isMounted = true;
+
+        // HACK: imperatively add an ID onto the Mathquill input
+        // (which in mobile is a span; desktop a textarea)
+        // in order to associate a visual label with it
+        if (this.refs.input) {
+            const isMobile = this.props.apiOptions.customKeypad;
+            const container = ReactDOM.findDOMNode(this.refs.input);
```

Comment:
Nit: This code is fine for right now, but just want to highlight that if it is at all possible to avoid using `findDOMNode` function as it's deprecated in future versions of React. This isn't a problem now as we're just moving to version 18. But wanted to call it out as a food for thought 😄